### PR TITLE
Ignore one more warning with GCC 8.x compilers

### PR DESCRIPTION
**Pull request by permcody**
The trigger comes from BOOST distributions

### CODE REVIEW

File: include/utils/ignore_warnings.h [hunk 1]
```diff
@@ -82,6 +82,9 @@
 #if (__GNUC__ > 6)
 #pragma GCC diagnostic ignored "-Wint-in-bool-context"
 #pragma GCC diagnostic ignored "-Wimplicit-fallthrough"
+#if (__GNUC__ > 8)
```
Comment (jwpeterson) [inline include/utils/ignore_warnings.h]: Does `> 8` mean GCC 9.x?

**Comment by permcody** [comment]
Oh no! That’s not right. This is for the GCC 8 compiler. The error went
away on my local test because I didn’t wrap it in the if statement. I’ll
fix it later unless you just want to do it. Nice catch!

**Comment by jwpeterson** [comment]
Sure, I can just change it to `> 7`. I can't test it conveniently since I don't have a GCC 8.x readily available, so you'll have to let me know if it still works.

**Comment by jwpeterson** [comment]
BTW, it looks like this warning has been around for a while (e.g. [it's mentioned in these GCC 3.4.4 docs](https://gcc.gnu.org/onlinedocs/gcc-3.4.4/gcc/Warning-Options.html)). I guess the ability to disable it was what was only recently added?

**Pull request merged**